sms/send-signed-sms.php

```diff
@@ -0,0 +1,16 @@
+<?php
+require_once __DIR__ . '/../config.php';
+require_once __DIR__ . '/../vendor/autoload.php';
+
+// add a 3rd parameter to change the signature from md5hash to something else
+$signed = new Nexmo\Client\Credentials\SignatureSecret(NEXMO_API_KEY, NEXMO_API_SIGNATURE_SECRET);
```

Comment:
Should we explicitly set the signature method here? I made it optional as making it mandatory would be a BC break but would like to encourage people to explicitly set it

Reply:
if you like? I was trying to make it simple in the use case but clear how to edit.

Reply:
Amended, do you want to take another look now?